taichi/transforms/offload.cpp

```diff
@@ -477,11 +479,11 @@ class FixCrossOffloadReferences : public BasicStmtVisitor {
  private:
   FixCrossOffloadReferences(
       const CompileConfig &config,
-      const StmtToOffsetMap &local_to_global_offset,
-      const std::unordered_map<Stmt *, Stmt *> &stmt_to_offloaded,
+      const StmtToOffsetMap *local_to_global_offset,
```

Comment:
can't we just pass the reference here just like `stmt_to_offloaded`?

Reply:
@lin-hitonami it's just trying to fail loudly if a temporary is passed in as const ref.

Reply:
eg. https://zpjiang.me/2020/01/20/const-reference-vs-pointer/